src/node.cc

```diff
@@ -315,27 +319,26 @@ MaybeLocal<Value> Environment::BootstrapInternalLoaders() {
 MaybeLocal<Value> Environment::BootstrapNode() {
   EscapableHandleScope scope(isolate_);
 
-  Local<Object> global = context()->Global();
-  // TODO(joyeecheung): this can be done in JS land now.
-  global->Set(context(), FIXED_ONE_BYTE_STRING(isolate_, "global"), global)
-      .Check();
-
   // process, require, internalBinding, isMainThread,
-  // ownsProcessState, primordials
+  // ownsProcessState, primordials, globalThis
   std::vector<Local<String>> node_params = {
       process_string(),
       require_string(),
       internal_binding_string(),
       FIXED_ONE_BYTE_STRING(isolate_, "isMainThread"),
       FIXED_ONE_BYTE_STRING(isolate_, "ownsProcessState"),
-      primordials_string()};
+      primordials_string(),
+      // TODO(joyeecheung): remove this once we only need to support V8
+      // that ships globalThis.
```

Comment:
I don't really understand this comment. The V8 we have in v12.x supports `globalThis`. Do you intend to cherry-pick this change to v10.x ?

Reply:
I guess my question is do we still need to backport as much as possible to v10.x now? It's still active right?

If we don't I can happily remove the polyfill and just use the globalThis from V8.

Reply:
I good compromise could be to add a second commit to this PR that removes the polyfill and only backport the first one.

Reply:
I was thinking about doing that in a follow-up PR because we can then use don-land labels.